chrupd.cmd

```diff
@@ -1,6 +1,6 @@
 <# :
 @echo off
-SETLOCAL & SET "PS_BAT_ARGS=%~dp0 %*"
+SETLOCAL & SET PS_BAT_ARGS="%~dp0" %*

```

Comment:
My mistake. This should read
`SETLOCAL & SET PS_BAT_ARGS="%~dp0 %*"` instead of `SETLOCAL & SET PS_BAT_ARGS="%~dp0" %*`

Reply:
Hi, 

Thanks for bringing this to my attention and for your suggested fix.
Indeed it should not have been a real problem since the first powershell code checks first if the dir exists and uses an alternative method to get the `$scriptDir` if it does not.

Since the Batch part also passes command line arguments (e.g. `chrupd.cmd -h`) I used a different method with pipe sign as separator to handle a path with spaces:

`SETLOCAL & SET "PS_BAT_ARGS=\""%~dp0|"\" %*"`
...which then gets removed in PS code:
`$Args = $Args -Replace '\|', ''`

Not the prettiest, but oh well ;)